docs/docs/spark-getting-started.md

```diff
@@ -48,8 +48,6 @@ This command creates a path-based catalog named `local` for tables under `$PWD/w
 ```sh
 spark-sql --packages org.apache.iceberg:iceberg-spark-runtime-3.5_2.12:{{ icebergVersion }}\
     --conf spark.sql.extensions=org.apache.iceberg.spark.extensions.IcebergSparkSessionExtensions \
-    --conf spark.sql.catalog.spark_catalog=org.apache.iceberg.spark.SparkSessionCatalog \
-    --conf spark.sql.catalog.spark_catalog.type=hive \
```

Comment:
I tried to do this 2 years ago and my PR didn't get merged. 

More info: https://github.com/apache/iceberg/pull/6250

Reply:
SGTM